content-accelerator/latest/using/user-guide.md

```diff
@@ -1154,6 +1155,65 @@ you to search for all possible documents in the repository.
 
 * The purple box labeled `6` is where you can submit your search form, executing the search. This can also be done by clicking the search button in the full text search input box or by hitting the enter key while within the form. Clicking the reset button empties all previous entered criteria in the form without running another search.
 
+### Advance Search capabilities with Attribute Search
+
+Attribute Search includes the option to configure Advanced Search capabilities within the search form. To enable advanced capabilities in attribute search, navigate to the Attribute Search config in the Alfresco Content Accelerator Admin. By default, the values are set to `Disabled`:
+
+To enable advanced capabilities in attribute search, complete the following steps.
+
+1. Navigate to the Attribute Search config in the Alfresco Content Accelerator Admin and set the value of the Show Advanced Search parameter to Enabled. By default, the value is set to Disabled.
+
+    ![Advanced Search configuration]({% link content-accelerator/images/aca-show-advanced-search-config.png %})
+
+2. After you enable Advanced Search, you can enable the following configuration settings:
+  * Enable Any/All Search
+  * Enable Like/Exact/Not Search
+
+    ![Advanced Search additional configuration]({% link content-accelerator/images/aca-show-advanced-search-options-config.png %})
+
+When the Enable Any/All Search setting is enabled, you can configure either of the following settings.
+
+| Settings | Search operator |
+| -------- | --------------- |
+| ANY of these values | If you select this option, search is performed based on the OR operator |
+| ALL of these values |	If you select this option, search is performed based on the AND operator |
+
```

Comment:
Please add.

If the **Enable Any/All disabled** setting is disabled, the default operation is the AND operation.